docs/pages.jl

```diff
@@ -1,24 +1,71 @@
-                                            "catalyst_functionality/parametric_stoichiometry.md",
-                                            "catalyst_functionality/network_analysis.md",
-                                            "catalyst_functionality/chemistry_related_functionality.md",
-                                            "Model creation examples" => Any["catalyst_functionality/example_networks/basic_CRN_examples.md",
-                                                                             "catalyst_functionality/example_networks/hodgkin_huxley_equation.md",
-                                                                             "catalyst_functionality/example_networks/smoluchowski_coagulation_equation.md"]],
-            "Catalyst Applications" => Any["catalyst_applications/simulation_structure_interfacing.md",
-                                           "catalyst_applications/advanced_simulations.md",
-                                           "catalyst_applications/homotopy_continuation.md",
-                                           "catalyst_applications/nonlinear_solve.md",
-                                           "catalyst_applications/bifurcation_diagrams.md"],
-            "Inverse Problems" => Any["inverse_problems/optimization_ode_param_fitting.md",
-                                        #"inverse_problems/petab_ode_param_fitting.md",
-                                        "inverse_problems/structural_identifiability.md",
-                                        "Inverse problem examples" => Any["inverse_problems/examples/ode_fitting_oscillation.md"]],
-            "FAQs" => "faqs.md",
-            "API" => "api/catalyst_api.md"]
\ No newline at end of file
+pages = Any[
+    "Home" => "home.md",
+    "Introduction to Catalyst" => Any[
+        "01_introduction_to_catalyst/01_catalyst_for_new_julia_users.md",
+        "01_introduction_to_catalyst/02_introduction_to_catalyst.md"
+        # Advanced introduction.
+    ],
+    "Model Creation" => Any[
+        "02_model_creation/01_dsl_description.md",
+        # DSL - Advanced
+        "02_model_creation/03_programmatic_CRN_construction.md",
+        "02_model_creation/04_constraint_equations.md",
+        # Events.
+        "02_model_creation/06_compositional_modeling.md",
+        # Distributed parameters, rates, and initial conditions.
+        # Loading and writing models to files.
+        "Model creation examples" => Any[
+            "02_model_creation/examples/basic_CRN_examples.md",
+            "02_model_creation/examples/hodgkin_huxley_equation.md",
+            "02_model_creation/examples/smoluchowski_coagulation_equation.md"
+        ]
+    ],
+    "Reaction network-related functionalities" => Any[
+        # Model visualisation.
+        "03_reaction_network_functionality/network_analysis.md",
+        "03_reaction_network_functionality/chemistry_related_functionality.md",
```

Comment:
I'm not sure I'd call the chemistry functionality network-related functionality. It doesn't really have anything to do with network properties.